Add keyboard support to the wizards

JeffreyDevloo opened the pull request:
Added a form on the modal generator
Removed forms from individual wizards
Added id option to the form that can be overwritten by every wizard using the self.id value. By default the value will be a generated hash
These changes add keyboard support to the wizards and fixes https://github.com/openvstorage/framework/issues/1053

JeffreyDevloo committed:
Added a form on the modal generator
Removed forms from individual wizards
Added id option to the form that can be overwritten by every wizard using the self.id value. By default the value will be a generated hash
These changes add keyboard support to the wizards and fixes https://github.com/openvstorage/framework/issues/1053

khenderick commented on webapps/frontend/app/views/wizards/createfromtemplate/gather.html:
```diff
@@ -16,7 +16,7 @@
 -->
 <div>
-    <form data-bind="visible: canStart().value" class="form-horizontal">
+    <div data-bind="visible: canStart().value" class="form-horizontal">
```
I think the `class="form-horizontal"` is not required anymore

khenderick commented on webapps/frontend/app/views/wizards/index.html:
```diff
@@ -17,29 +17,41 @@
+            </div>
+            <div data-bind="css: { 'modal-footer': modal() }">
+                <div class="pull-left">
+                    <button type='button' class="btn btn-default" name="cancel"
```
Please use double quotes.

khenderick commented on webapps/frontend/app/views/wizards/index.html:
```diff
@@ -17,29 +17,41 @@
+                    </button>
+                    <button class="btn btn-primary" name="next"
+                            data-bind="attr: {type: (canNext() == true ? 'submit' : 'button')}, css: { disabled: !canNext() || loadingNext() },
+                            visible: hasNext(), click: next, disable: !canNext()" data-i18n="[html]ovs:generic.next">
```
Messy alignment

JeffreyDevloo committed:
Review fixes:
- Fixed alignment (now aligning to the first attribute value instead of attribute
- Replaced single quotes with double quotes for consistency
- Removed unnecessary styling on certain divs
